control-plane/agents/src/bin/core/node/wrapper.rs

```diff
@@ -1384,19 +1384,26 @@ impl NexusChildActionApi<NexusChildActionContextNode> for Arc<tokio::sync::RwLoc
 impl NexusSnapshotApi for Arc<tokio::sync::RwLock<NodeWrapper>> {
     async fn create_nexus_snapshot(
         &self,
-        _request: &CreateNexusSnapshot,
+        request: &CreateNexusSnapshot,
     ) -> Result<CreateNexusSnapshotResp, SvcError> {
-        todo!()
+        let dataplane = self.grpc_client_locked(request.id()).await?;
+        dataplane.create_nexus_snapshot(request).await
     }
 }
 
 #[async_trait]
 impl ReplicaSnapshotApi for Arc<tokio::sync::RwLock<NodeWrapper>> {
     async fn create_repl_snapshot(
         &self,
-        _request: &CreateReplicaSnapshot,
+        request: &CreateReplicaSnapshot,
     ) -> Result<CreateReplicaSnapshotResp, SvcError> {
-        todo!()
+        let dataplane = self.grpc_client_locked(request.id()).await?;
```

Comment:
this will clash with my PR, merging mine now, please rebase and resolve.

Reply:
rebased and updated.